synapse/events/utils.py

```diff
@@ -398,7 +398,7 @@ async def serialize_event(
         """Serializes a single event.
 
         Args:
-            event
+            event: The event being serialized.
             time_now: The current time in milliseconds
             bundle_aggregations: Whether to bundle in related events
```

Comment:
I... don't know what this means.

(also: are bundled relations the same as bundled aggregations?)

Reply:
> (also: are bundled relations the same as bundled aggregations?)

I believe so, the MSC is really confusing at what it calls bundled relations & aggregations.